p/path-loader/path-loader_ubi_8.3.sh

```diff
@@ -0,0 +1,51 @@
+# -----------------------------------------------------------------------------
+#
+# Package	    : path-loader
+# Version	    : v1.0.10
+# Source repo	: https://github.com/whitlockjc/path-loader.git
+# Tested on	    : UBI 8.3
+# Language      : Node
+# Travis-Check  : True
+# Script License: Apache License, Version 2 or later
+# Maintainer	: Srividya Chittiboina <Srividya.Chittiboina@ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+set -e
+
+WORK_DIR=`pwd`
+
+PACKAGE_NAME=path-loader
+PACKAGE_VERSION=${1:-v1.0.10}
+PACKAGE_URL=https://github.com/whitlockjc/path-loader.git
+
+yum install git -y
+
+#install phantomjs
+yum install wget bzip2 freetype fontconfig -y
+wget https://github.com/ibmsoe/phantomjs/releases/download/2.1.1/phantomjs-2.1.1-linux-ppc64.tar.bz2
+tar -xvf phantomjs-2.1.1-linux-ppc64.tar.bz2
+ln -s /phantomjs-2.1.1-linux-ppc64/bin/phantomjs /usr/local/bin/phantomjs
+export PATH=$PATH:/phantomjs-2.1.1-linux-ppc64/bin/phantomjs
```

Comment:
Do we need this PATH export, as we created link to phatomjs as `/usr/local/bin/phantomjs` ?
If we need it also, we need to add the directory to the PATH variable, not binary.